src/MembershipsSDKAdapter.js

```diff
@@ -0,0 +1,64 @@
+import {BehaviorSubject, throwError} from 'rxjs';
+import {DestinationType, MembershipsAdapter} from '@webex/component-adapter-interfaces';
+
+export default class MembershipsSDKAdapter extends MembershipsAdapter {
+  /**
+   * Gets the active members in a meeting
+   *
+   * @static
+   * @param {object} members Members object from meeting, keyed by ID
+   * @returns {Array}
+   * @memberof MembershipsSDKAdapter
+   */
+  static getActiveMembers(members) {
+    return Object.values(members)
+      .filter((member) => !member.isInMeeting)
+      .map((member) => ({
+        id: member.id,
+      }));
+  }
+
+  /**
+   * Returns an observable that emits a Membership object.
+   * Whenever there is an update to the membership, the observable
+   * will emit a new updated Membership object, if datasource permits.
+   *
+   * @param {string} destinationID  ID of the destination for which to get members
+   * @param {DestinationType} destinationType  type of the membership destination
+   * @returns {external:Observable.<Membership>} Observable stream that emits membership data
+   * @memberof MembershipAdapter
+   */
+  getMembersFromDestination(destinationID, destinationType) {
+    if (destinationType !== DestinationType.MEETING) {
+      return throwError(new Error(`getMembersFromDestination for ${destinationType} is not currently supported.`));
+    }
+
+    const meeting = this.datasource.meetings.getMeetingByType('id', destinationID);
+
+    if (!meeting) {
+      return throwError(new Error(`Meeting ${destinationID} not found in datasource.`));
+    }
+
+    // Behavior subject will keep the last emitted object for new subscribers
+    // https://rxjs.dev/guide/subject#behaviorsubject
+    const membershipSubject = new BehaviorSubject([]);
```

Comment:
What is the value in making this a subject? In the current implementation whenever `getMembersFromDestination` gets called, a new subject is going to be created so there will never be multiple subscribers.
I think using the subject itself its fine, but how are you making sure that whenever there is a call for the same ID + Type combo the same subject is reused?